torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -41,52 +41,71 @@ const std::string shape_compute_functions =
           assert len(out) == 2
           return [out[0], out[1]]
 
+        # TODO: maybe make it customary that extra arguments are unused ?
+        # TODO: return self directly
+        def unary_two_unused_inputs(self: List[int], inp0: Any, inp1: Any):
```

Comment:
Just trying to understand the bigger picture here, does mean we need unary_one_unused_inputs, unary_three_unused_inputs, etc.? If those are unused inputs, why do we need to spell them out?

Reply:
Currently our symbolic shape analysis expects the number of inputs in the shape compute graph for an operator to have the same number of inputs as the operator. I think in a follow up i might make it so that it is implicit that extra arguments are unused.